docs/CHANGELOG.asciidoc

```diff
@@ -70,6 +71,98 @@ Review important information about the {kib} 7.16.x releases.
+* Adds event loop delay histogram percentiles to `metrics.ops` logger message and meta field {kibana-pull}120451[#120451]
+
+[float]
+[[fixes-v7.17.0]]
+==== Bug Fixes
+Alerting::
+Fixes call to update index alias to hidden {kibana-pull}122882[#122882]
+Canvas::
+* Fixes an issue where a workpad element was breaking after changing the expression from expression editor and resizing {kibana-pull}122154[#122154]
+* Repeat image bug with image height fixed {kibana-pull}121497[#121497]
+Discover::
+* Fixes saved search hidden chart can't be opened when returning to Discover {kibana-pull}122745[#122745]
+* Fixes page up and down navigation in classic table {kibana-pull}122742[#122742]
+* Fixes context view for document ids containing special characters {kibana-pull}122737[#122737]
+* Adds {kib} services provider for embeddable {kibana-pull}121621[#121621]
+Elastic Security::
+For the Elastic Security 7.17.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Lens & Visualizations::
+* Restores default chart `timezone` to `'local'` in XY charts {kibana-pull}123264[#123264]
+* Fixes y-axis visibility for Percentile aggregation in XY charts {kibana-pull}122162[#122162]
+* Runs the migration correctly for the drop last bucket in *TSVB* {kibana-pull}121734[#121734]
+* Fixes percentage mode behavior when multiple metrics are placed on the same axis in XY charts {kibana-pull}121610[#121610]
+* "Bucket script" and "Series Agg" should be active for "timerange" mode in *TSVB* {kibana-pull}121301[#121301]
+Machine Learning::
+Fixes Anomaly detection's custom url with special characters {kibana-pull}121411[#121411]
+Management::
+* Fixes runtime field validations not supporting special script types {kibana-pull}122469[#122469]
+* Allows fields named `_timestamp` to be non-date types {kibana-pull}121772[#121772]
+Maps::
+Fixes map application crashes when leaving map while drawing filter {kibana-pull}122353[#122353]
+Monitoring::
+Fixes telemetry for cases attached to alerts to work across spaces {kibana-pull}122477[#122477]
+Observability::
+* Fixes Uptime alerting fly outs within alerting app {kibana-pull}123031[#123031]
+* Overview link to Exploratory View should not filter by monitor in Uptime {kibana-pull}122907[#122907]
+* Adds a global filter to exploratory view when there is only one series in Uptime {kibana-pull}122301[#122301]
+* Fixes a problem in which the exploratory view would crash if no date ranges were selected in Uptime {kibana-pull}122163[#122163]
+* Fixes a bug where the monitor step details would send more requests than necessary in Uptime {kibana-pull}121889[#121889]
+* Drop requests for overview page data when filter not initialized in Uptime {kibana-pull}120031[#120031]
+Platform::
+Displays event loop delay metrics in the Status page to assist with performance monitoring {kibana-pull}121052[#121052]
```

Comment:
This wasn't a bug -- probably should be put under enhancements by the other event loop delay item.